spk/homeassistant/src/installer.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+
+# Package
+PACKAGE="homeassistant"
+DNAME="Home Assistant"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PYTHON_DIR="/usr/local/python3"
+PATH="${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+USER="homeassistant"
+GROUP="users"
```

Comment:
No, it probably won't then. If that's a useable option, you might want to take a look at #1649 and add that approach in as well.
How can people set that downloadpath, is it available via the interface?